vllm/model_executor/models/utils.py

```diff
@@ -279,3 +280,18 @@ def is_pp_missing_parameter(name: str, model: torch.nn.Module) -> bool:
         if name.startswith(missing_layer_name):
             return True
     return False
+
+def make_empty_intermediate_tensors_factory(keys: List[str],
+                                            hidden_size: int) -> Callable:
```

Comment:
```suggestion
                                            hidden_size: int):
```

You lose type information by using `Callable` without type arguments. Instead, just let the type checker automatically infer the type of the returned function.

Reply:
Done